app/svg.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
1/5 move to package `utils`?

Reply:
3/5 but `utils` needs to die a painful death. :D

Reply:
Sure thing. I do wonder if app/file and/or app/image would make sense then? I do question `app` as the place for this function.

Reply:
I guess this fits into the ongoing discussion on go package organization. I don't see a monolithic `app` /package/ as necessarily a problem (`net/http` is huge!) -- rather, it's the monolithic `App` struct that causes most of our grief.

This function is an unexported, internal implementation detail connected with the existing file handling operations in the `app` package, and probably belongs close to them. If we can come up with an abstraction for file handling (that isn't just tightly coupled back to `App` from outside the package), it would definitely make sense to go there.

Reply:
Ok.